packages/workbox-precaching/utils/generateUrlVariations.mjs

```diff
@@ -0,0 +1,55 @@
+/*
+  Copyright 2019 Google LLC
+
+  Use of this source code is governed by an MIT-style
+  license that can be found in the LICENSE file or at
+  https://opensource.org/licenses/MIT.
+*/
+
+import removeIgnoredSearchParams from './removeIgnoredSearchParams.mjs';
+
+import '../_version.mjs';
+
+/**
+ * Generator function that yields possible variations on the original URL to
+ * check, one at a time.
+ *
+ * @param {string} url
+ * @param {Object} options
+ *
+ * @private
+ * @memberof module:workbox-precaching
+ */
+export default function* (url, {
```

Comment:
Same comment as above, can we make this a named export?

Reply:
Changed.